.github/workflows/build_python_wheels_macos.yml

```diff
@@ -43,6 +44,7 @@ jobs:
         - { version: 3.6, include_dir_name: python3.6m/}
         - { version: 3.7, include_dir_name: python3.7m/}
```

Comment:
why do some end on m and others dont?

Reply:
Not 100% sure but it changed over the course of the releases

Reply:
the `m` suffix indicates its built using a pymalloc allocator. In most cases i've seen its the same thing (3.6 symlinks to 3.6m), but technically the libraries are not guaranteed to be binary compatible